llpc/context/llpcCompiler.cpp

```diff
@@ -826,6 +830,7 @@ Result Compiler::buildPipelineWithRelocatableElf(Context *context, ArrayRef<cons
       // Release Entry
       ReleaseCacheEntry(false, nullptr, &cacheEntry);
       LLPC_OUTS("Cache hit for shader stage " << getShaderStageName(static_cast<ShaderStage>(stage)) << "\n");
+      stageCacheAccesses[stage] = CacheAccessInfo::CacheHit;
```

Comment:
You probably want to assert somewhere near the top of the loop that `stageCacheAccess` has enough elements to avoid potential OOB errors if the stages change in the future.

Reply:
done